### DISCUSSION TITLE
Field组件在这里相当于其他UI库的FormItem吗？

### DISCUSSION THREAD

**Discussion by popring** [youzan/vant#11166] [General]
有点不太理解最初这么设计是因为什么，这样 Field 这个组件就会包含其他很多逻辑，也就承载着更多功能，输入框和FormItem这两个功能就进行了耦合，也会让开发者有点迷惑，哪些功能是输入框的特有功能，哪些是可以脱离输入框自定义组件使用的。

希望作者出来聊一下，哈哈哈，或者有懂哥，来聊聊。

**Comment by chenjiahan** [comment]
确实目前 FormItem 是使用 Field 组件实现的，这个主要还是历史原因 —— Vant 一开始只有 Field 组件，没有 Form 组件。后来在迭代过程中引入了 Form 组件，并把 Field 组件作为了 Form 推荐的子组件进行处理，于是就变成了这种使用方式。

当然，后来我们也引入了其他解决方案，就是 [useCustomFieldValue](https://vant-ui.github.io/vant/#/zh-CN/use-custom-field-value) 这个 hook，通过这个 hook 可以把任意组件变成一个 FormItem 来使用。

**Reply by popring** [reply]
useCoustomFieldValue 能解决值的问题，但如果这里也需要使用自带的校验规则，还是需要将值传进 Field 组件使用吗？

**Reply by chenjiahan** [reply]
不应该把值再传进 Field 组件，目前自定义表单组件的校验规则确实没有提供解决方案，我们后续评估下

**Reply by popring** [reply]
好的，谢谢回复。👍🏻
